packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/OutputTypes/Parameter.cs

```diff
@@ -3,67 +3,94 @@
 
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
 using Microsoft.Generator.CSharp.Expressions;
 using Microsoft.Generator.CSharp.Input;
 
 namespace Microsoft.Generator.CSharp
 {
-    public sealed record Parameter(string Name, FormattableString? Description, CSharpType Type, ValueExpression? DefaultValue, ValidationType Validation, ValueExpression? Initializer, bool IsApiVersionParameter = false, bool IsEndpoint = false, bool IsResourceIdentifier = false, bool SkipUrlEncoding = false, RequestLocation RequestLocation = RequestLocation.None, SerializationFormat SerializationFormat = SerializationFormat.Default, bool IsPropertyBag = false, bool IsRef = false, bool IsOut = false)
+    [DebuggerDisplay("{GetDebuggerDisplay(),nq}")]
+    public sealed class Parameter
     {
-        internal bool IsRawData { get; init; }
+        public string Name { get; }
+        public FormattableString Description { get; }
+        public CSharpType Type { get; init; }
+        public ValueExpression? DefaultValue { get; init; }
+        public ValidationType? Validation { get; init; } = ValidationType.None;
+        public ValueExpression? Initializer { get; init; }
```

Comment:
this property shouldn't be needed since you will add any initializer statements to whatever method body needs them.